README.md

```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
 [nREPL]: https://github.com/clojure/tools.nrepl
 [Arcadia Unity]: https://github.com/arcadia-unity/Arcadia
 
-## How to use
-TBD
+### How to use Cursive with Arcadia
+
+In your Unity project with Arcadia, do:
+
+1. Create a new "regular" looking clojure project that has a deps.edn at its root. For example:
+
+  ```sh
+cd Assets
+mkdir <project-name>
+cd <project-name>
+mkdir src test target
+echo '{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
```

Comment:
Do we need to import this if we're working with Unity?

Reply:
We still want a clojure version defined which Cursive thinks we're going to use. It'll use this clojure version to evaluate the code we write.

For reasons mentioned in https://github.com/eponai/tools.cursive-arcadia-repl/pull/5#discussion_r285393034 I still think we should **explicitly** define the clojure version used.